src/components/Footer.tsx

```diff
@@ -1,66 +1,47 @@
-import {
-  Box,
-  Center,
-  Container,
-  HStack,
-  Link,
-  Stack,
-  useColorModeValue,
-} from '@chakra-ui/react'
+import { Box, Container, Link, Text, VStack } from '@chakra-ui/react'
 import Image from 'next/image'
 
 import poweredByVercelLogo from '@app-public/powered-by-vercel.svg'
-import { REPO_URL } from '~/common'
 
 const Footer = () => {
-  const boxBgColor = useColorModeValue('gray.50', 'gray.900')
-
   return (
-    <Box as="footer" bg={boxBgColor} flexShrink={0}>
-      <Container py={5} variant="fluid">
-        <Stack
-          justify="space-between"
-          direction={['column', 'row']}
-          spacing={4}
-        >
-          <HStack
-            shouldWrapChildren
-            justify="space-between"
-            alignItems="center"
-            justifyContent="center"
-            fontSize={{ base: 'md', md: 'lg' }}
+    <Box as="footer" bgColor="background2">
+      <Container
+        centerContent
+        maxWidth="full"
+        py={{ base: 1, lg: 4 }}
+        px={{ base: 4, lg: 5 }}
+      >
+        <VStack spacing={5}>
+          <Text
+            color="primaryText"
+            fontSize={{ base: 'lg', md: '2xl' }}
+            fontWeight="black"
+            textAlign="center"
           >
-            <Box>
-              Created with{' '}
-              <span role="img" aria-label="Love">
-                💜
-              </span>{' '}
-              by{' '}
-              <Link isExternal href="https://mario.dev/">
-                Mario
-              </Link>
-            </Box>
-            <Box as="span">-</Box>
+            Created with love <br /> by{' '}
```

Comment:
I like the new footer! Now we both get credit for our work. ❤️ 

I think we need a bit more empty space between the Vercel badge bottom and the page end. On Figma there's `48px` worth of empty space.

Reply:
I probably mixed up the spacing there. I'll fix it later.